src/WebJobs.Script/Content/Script/functions.js

```diff
@@ -88,22 +88,19 @@ function getEntryPoint(f, context) {
         }
         else if (Object.keys(f).length == 1) {
             // a single named function was exported
-            f = f[Object.keys(f)[0]];
+            var name = Object.keys(f)[0];
+            f = f[name];
         }
         else {
             // finally, see if there is an exported function named
             // 'run' or 'index' by convention
-            f = f['run'] || f['index'];
+            f = f.run || f.index;
         }
     }
-    else if (!util.isFunction(f)) {
-        // the module must export an object or a function
-        f = null;
-    }
 
-    if (!f) {
```

Comment:
Previously we were only checking if f was not null, not that it was a function. E.g. in multiple export cases, we previously would have allowed { run: 1 } and tried to use 1 as a function.